numpy/core/tests/test_scalarbuffer.py

```diff
@@ -0,0 +1,94 @@
+"""
+Test scalar buffer interface adheres to PEP 3118
+"""
+import sys
+import numpy as np
+from numpy.testing import run_module_suite, assert_, assert_equal, dec
+
+# types
+scalars = [
+    np.bool_,
+    np.byte, np.short, np.intc, np.int_, np.longlong,
+    np.ubyte, np.ushort, np.uintc, np.uint, np.longlong,
+    np.half, np.single, np.double, np.longdouble,
+    np.csingle, np.cdouble, np.clongdouble,
+]
+
+# PEP3118 format strings for native (standard alignment and byteorder) types
+scalars_set_code = [
+    (np.bool_, '?'),
+    (np.byte, 'b'),
+    (np.short, 'h'),
+    (np.intc, 'i'),
+    (np.int_, 'l'),
+    (np.longlong, 'q'),
+    (np.ubyte, 'B'),
+    (np.ushort, 'H'),
+    (np.uintc, 'I'),
+    (np.uint, 'L'),
+    (np.ulonglong, 'Q'),
+    (np.half, 'e'),
+    (np.single, 'f'),
+    (np.double, 'd'),
+    (np.float_, 'd'),
```

Comment:
These two are aliases, so there's no point having both - just use `np.double`, and ditch `np.float_`